index.html

```diff
@@ -111,6 +111,18 @@ <h2>
 		</td>
 	      </tr>
 
+	      <tr>
+		<td style="vertical-align:top;height:4em">
+		  <a class="btn btn-success" href="cite.html">Cite! &raquo;</a>
```

Comment:
The other green boxes all have a different size. I think that's because they have the  `style="width:100%" ` code? Can you fix that?